extra-multimedia/feeluown-qqmusic/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME="feeluown-qqmusic"
+PKGSEC=sound
+PKGDES="a FeelUOwn plugin for QQMusic"
```

Comment:
First letter in `PKGDES=` should be capitalised; the "A" is actually not necessary here.